CPT331.Data/TestRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿#region Using References
+
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+
+using Dapper;
+
+#endregion
+
+namespace CPT331.Data
+{
+	public static class TestRepository
+	{
+		public static List<Test> GetTests()
+		{
+			List<Test> getTests = null;
+
+			using (SqlConnection sqlConnection = SqlConnectionFactory.NewSqlConnetion())
+			{
+				getTests = SqlMapper
```

Comment:
Seems simple enough. SPs look like a good way to avoid to much querying in code (which is difficult to test/maintain).